### ISSUE TITLE
OLGLMM.family not found

### ISSUE THREAD

**Issue by rossdwilliams** [ohdsi-studies/OLGLMM-COVID#2]
When running:
```
olglmmCovid:::executeProspective(
  databaseDetails = databaseDetails,
  siteId = siteId,
  outputFolder = outputFolder,
  createCohorts = F, 
  cohortTable = cohortTable,
  createData = F, 
  createPlot = F, 
  run = T,
  verbosity = "INFO"
        )
```

I get the following error 

```
Error in get(paste0(control$model, ".family"))  :
  object 'OLGLMM.family' not found
```

**Comment by jreps** [comment]
please ensure the latest pda is installed by running remotes::install_github('penncil/pda') , making sure it installs and then restarting R.

**Comment by jreps** [comment]
also make sure you have the latest study package by doing the same for remotes::install_github('ohdsi-studies/OLGLMM-COVID')

**Comment by rossdwilliams** [comment]
I am running pda v1.2.6 and then olglmm with 9c616c8. (says its v0.0.1 but think thats because its unreleased.)

I still have the same issue, i also get a warning that my project_name suggests incorrect wave? and no cloud uri found

**Comment by jreps** [comment]
Odd - I have v1.2.5 and it has OLGLMM.family, so any later versions should also have it.  

Are you running inside an renv?  if so, try installing pda and the package in a normal session to see whether it was renv messing up the versions.

You can ignore the project_name warning, it happens because the json project name is not omicron_1

**Comment by jreps** [comment]
Also, you can test the package will work by running pda:::OLGLMM.family and seeing whether it returns "binomial"

**Comment by rossdwilliams** [comment]
I am not running inside renv as i was having install problems with not being on an internet enabled machine. So all my installs are done in the classic way. 
```
 > pda:::OLGLMM.family
Error: object 'OLGLMM.family' not found
```

when typing it i get lots of options for that are exported but i dont see any of the internal functions.

**Comment by jreps** [comment]
ok - I think I realize the difference.  I've been using the GitHub version, that is 1.2.5 .  I don't know how you have version 1.2.6  since 1.2.5 is the version in the GitHub description https://github.com/Penncil/pda/blob/master/DESCRIPTION .  The version you are using may have a bug.

Can you try to use the remotes::install_github as that should install the 1.2.5 version (or use the 1.2.5 release).  Hopefully pda:::OLGLMM.family will then work once you have v1.2.5

**Comment by rossdwilliams** [comment]
Interesting, i used install.packages("pda"). I will revert it back to 1.2.5when im in tomorrow afternoon! 

If i look at the description in the github i see 1.2.5 but if i look at the releases on the landing page i see 1.2.5

**Comment by rossdwilliams** [comment]
I just round to trying this again, sorry for the delay.

with pda v1.2-5 i still get the above issue

if i try 
```R
pda:::OLGLMM.family()
```

i get an error, I figured out what the issue with this is. The OLGLMM are not available in either 1.2-5 or 1.2-6 release (so when i CRAN installed and when i took the release zip I didnt get those functions)

1.2-5
![image](https://github.com/ohdsi-studies/OLGLMM-COVID/assets/34747165/737c9303-a70c-4538-959d-3ffc3c37c146)

1.2-6
![image](https://github.com/ohdsi-studies/OLGLMM-COVID/assets/34747165/54a29a19-2a09-499c-bec2-b795ce285a23)

When i just pulled down a zip from main it then worked.
